libkbfs/folder_branch_ops.go

```diff
@@ -2135,7 +2149,7 @@ func (fbo *folderBranchOps) createEntryLocked(
 }
 
 func (fbo *folderBranchOps) doMDWriteWithRetry(ctx context.Context,
-	lState *lockState, fn func(lState *lockState) error) error {
+	lState *lockState, waitForCR bool, fn func(lState *lockState) error) error {
```

Comment:
@strib Do you think this is OK? Although not many functions use `doMDWriteWithRetry` directly so it's probably fine. But I started to wonder if makes sense to put Excl flag into ctx?